SingularityUI/app/components/logs/TaskGroupContainer.jsx

```diff
@@ -46,17 +71,21 @@ let mapStateToProps = function (state, ownProps) {
       terminated: false
     };
   }
-  let taskGroup = state.taskGroups[ownProps.taskGroupId];
-  let tasks = taskGroup.taskIds.map(taskId => state.tasks[taskId]);
+  const taskGroup = state.taskGroups[ownProps.taskGroupId];
+  const tasks = taskGroup.taskIds.map(taskId => state.tasks[taskId]);
 
   return {
     initialDataLoaded: _.all(_.pluck(tasks, 'initialDataLoaded')),
     logDataLoaded: _.all(_.pluck(tasks, 'logDataLoaded')),
     fileExists: _.any(_.pluck(tasks, 'exists')),
+    finishedLogExists: _.any(_.pluck(tasks, 'taskFinishedLogExists')),
     terminated: _.all(_.pluck(tasks, 'terminated')),
-    path: state.path
+    path: state.path,
+    taskIds: taskGroup.taskIds
   };
 };
 
-export default connect(mapStateToProps)(TaskGroupContainer);
+const mapDispatchToProps = { doesFinishedLogExist };
```

Comment:
We haven't been using this pattern for dispatches so far, I didn't realize that you could do this. Should we stick with the pattern we've been using (mapDispatchToProps as a function that takes in `(dispatch, ownProps)`) or is this better for some reason?